dist/index.html

```diff
@@ -88,13 +88,34 @@ <h2 class="modal__title" id="modal-1-title">
         </main>
         <main class="modal__content modal-1-window-3 hidden" id="modal-1-content">
           <div>
-            <input id='tripDuration'>Trip Duration in Days</input>
+            <p>How many days will you be traveling?</p>
+            <select name="trip-duration" id="tripDuration">
```

Comment:
Have you considered an `input type="number"` for these? I dunno if it's optimal, necessarily, but what if someone wants to book a two-week trip?